backend/api/tests/test_views.py

```
@@ -0,0 +1,113 @@
+import pytest
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+from api.models import User, Video
+
+# Mark all tests in this module to use the database
+pytestmark = pytest.mark.django_db
+
+@pytest.fixture
+def api_client():
```

Ruff (0.8.2)

Missing return type annotation for public function `api_client`

(ANN201)